pyproject.toml

```diff
@@ -134,6 +134,9 @@ module = [
     "forex_python.converter",
     "login_required",
     "storages.backends.s3",
-    "social_django.models"
+    "social_django.models",
+    "emails",
+    "step_functions",
```

Comment:
Why are we adding these to an override though, shouldn't we just slowly fix the errors rather than just ignore them? For django_components that makes sense but internal code we should fix rather than hide (in my opinion)

Reply:
I'm adding them because this part I would like to leave on the end. Creating stubs for all this modules will take some time and I thought that it will be better to first have other errors cleared since they are less troublesome and numbers will go down quicker to acceptable level.

Reply:
Okay cool